nbs/05_readers.ipynb

```diff
@@ -1529,7 +1529,7 @@
     }
    ],
    "source": [
-    "sample_actiwatch.plot_light_activity()"
+    "# sample_actiwatch.plot_light_activity()"
```

Comment:
Let's add these back in!

Reply:
I think this will be added back when we merge this PR with #7, right? I might be confused though as to how the merge of both PRs will happen.

Reply:
I think I managed to rebase all the commits on this PR on the new main branch after the merge of #7